<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="portName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="conectPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writePort">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_stop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_go">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_back">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_left">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="robot_right">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>78</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
